.github/workflows/test_dbt_fal.yml

```diff
@@ -0,0 +1,155 @@
+name: Test dbt-fal adapter
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    paths-ignore:
+      - "examples/**"
+      - "docsite/**"
+      - "README.md"
+      - "LICENSE"
+      - "docker/**"
+
+  push:
+    branches: [act1]
```

Comment:
Should we now change this to only run if something changes in paths `adapter/**`?

Reply:
yes!